pom.xml

```diff
@@ -17,7 +17,7 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 
     <modelVersion>4.0.0</modelVersion>
-    <version>0.2.3-SNAPSHOT</version>
+    <version>0.3.1-SNAPSHOT</version>
```

Comment:
is it 0.3.0 or 0.3.1 ?

Reply:
0.3.1

Reply:
Current release=0.3.0, next snapshot version=0.3.1-SNAPSHOT